src/tests/api/HappyPath.test.ts

```diff
@@ -15,14 +15,86 @@ describe("Happy path tests", () => {
 			const sessionResponse = await startStubServiceAndReturnSessionId(journeyType);
 			const sessionId = sessionResponse.data.session_id;
 
-			const savedJourney = getKeyFromSession(sessionId, constants.DEV_CIC_SESSION_TABLE_NAME, "journey");
-			await expect(savedJourney).resolves.toBe(journeyType);
+			await getSessionAndVerifyKey(sessionId, constants.DEV_CIC_SESSION_TABLE_NAME, "journey", journeyType);
+			await getSessionAndVerifyKey(sessionId, constants.DEV_CIC_SESSION_TABLE_NAME, "authSessionState", "CIC_SESSION_CREATED");
 
 			const allTxmaEventBodies = await getTxmaEventsFromTestHarness(sessionId, 1);
 			validateTxMAEventData({ eventName: "CIC_CRI_START", schemaName }, allTxmaEventBodies);
 		});
 	});
 
+	describe("/claimedIdentity endpoint", () => {
+		it.each([
+			{ journeyType: "FACE_TO_FACE", schemaName: "CIC_CRI_START_SCHEMA" },
+			{ journeyType: "NO_PHOTO_ID", schemaName: "CIC_CRI_START_BANK_ACCOUNT_SCHEMA" },
+		])("Successful Request Tests - $journeyType", async ({ journeyType, schemaName }: { journeyType: string; schemaName: string }) => {
+			const sessionResponse = await startStubServiceAndReturnSessionId(journeyType);
+			const sessionId = sessionResponse.data.session_id;
+			const claimedIdentityResponse = await claimedIdentityPost(userData.firstName, userData.lastName, userData.dateOfBirth, sessionId);
+			expect(claimedIdentityResponse.status).toBe(200);
+
+			await getSessionAndVerifyKey(sessionId, constants.DEV_CIC_SESSION_TABLE_NAME, "authSessionState", "CIC_DATA_RECEIVED");
```

Comment:
Feel free to say that this is out of scope it's just something I've been thinking about for a while and since we're doing a refactor anyway was wondering if we can sneak it in 😅 

We currently hard code `DEV_CIC_SESSION_TABLE_NAME` in the constants file to be `session-cic-cri-ddb` - what do you think about also reading that from an env var? It's definitely possible that someone would have their own DB stack deployed

Reply:
I definitely agree with this. 
I've deployed a custom stack with template.yml updates to output the sessionTableName and make sure we're using that in the tests.

custom BE stack: cic-cri-api-1742

Test results: 
![image](https://github.com/govuk-one-login/ipv-cri-cic-api/assets/110032361/7c45d374-8a5e-45b6-8408-ed37cb35a747)